js/index.js

```diff
@@ -229,6 +229,18 @@ var WWW = (function(undefined) {
         if (ct.startsWith("text/")) {
             $('#content').val(data);
             return;
+        } else if (ct.startsWith("image/")) {
+          var link = $('<a></a>')
+            .attr('href', 'https://ptpb.pw/' + id)
+            .attr('target', '_blank')
+            .attr('title', 'Opens in a new window');
```

Comment:
Creating elements on the fly is ok, but when they contain several attributes like this that have the same value, I feel like copying from a pre-made `<a>` in `#stash` is better.